frame/treasury/src/lib.rs

```diff
@@ -346,6 +346,10 @@ pub mod pallet {
 		/// - O(1)
 		#[pallet::call_index(0)]
 		#[pallet::weight(T::WeightInfo::propose_spend())]
+		#[allow(deprecated)]
```

Comment:
wonder if developers will notice these deprecations, may be there is a better way to deprecate dispatchables?
this at least will be included in release notes

Reply:
You mean the pr will be included? Otherwise no one really will see this from the upper layers.

Reply:
yes, which is a good part. 
but combination of `#[allow(deprecated)]` and `#[deprecated(..)]` looks hacky to me, but otherwise CI fails because of the warnings. and this seams to be a general approach for deprecating dispatchable functions.
may be a solution could be to white list those warnings.

Reply:
@bkchr yes, which is a good part.
the combination of `#[allow(deprecated)]` and `#[deprecated(...)]` for one item looks hacky.
may be whitelisting those warnings could be a solution.

@ggwpez just checked, it does not. you can see it in rococo contracts, the contracts pallet has deprecated calls.
but it does include docs, so we could include it into docs.

Reply:
https://github.com/paritytech/substrate/issues/13976#issuecomment-1570081315

Reply:
I created a deprecation issue for it - https://github.com/paritytech/substrate/issues/14732
Can you please review the PR
I did not add anything to metadata since there is no agreement on it yet.